extensionsdk/Microsoft.Windows.DevHome.SDK/Microsoft.Windows.DevHome.SDK.idl

```diff
@@ -226,7 +226,7 @@ namespace Microsoft.Windows.DevHome.SDK
 
         ProviderOperationResult LogoutDeveloperId(IDeveloperId developerId);
 
-        event Windows.Foundation.TypedEventHandler<IDeveloperIdProvider, IDeveloperId> Changed;
+        event Windows.Foundation.TypedEventHandler<IDeveloperIdProvider> Changed;
```

Comment:
This was likely an unintentional change?

Reply:
Yep.  Changed back.